Lib/importlib/util.py

```diff
@@ -246,3 +247,13 @@ def exec_module(self, module):
         loader_state['__class__'] = module.__class__
         module.__spec__.loader_state = loader_state
         module.__class__ = _LazyModule
+
+
+def load_source(module_name, filename):
```

Comment:
The name is misleading because you're not loading from source, but from a file that contains source code. I think you're after something like `load_source_path()`.

You're also missing whether this is a package? If that's not possible for simplicity, then that should probably be stated. Otherwise it's a piece of information that a module spec contains.